src/browser/modules/EditorInfo/AddProperty.jsx

```diff
@@ -149,7 +149,11 @@ function AddProperty (props) {
       valueInput = (
         <TextInput
           id='propValue'
-          value={valueProperties || null}
+          value={
+            props.ToDisplay
+              ? valueProperties || null
+              : myState.newProperties.propValue
```

Comment:
stay with one state value, either valueProperties or myState.newProperties.propValue

i am not sure with which one to stay.